src/main.rs

```diff
@@ -211,6 +211,20 @@ fn redirect(dest: redirect::Destination) -> Redirect {
     Redirect::permanent(dest.uri)
 }
 
+#[get("/pdfs/<dest>")]
```

Comment:
Maybe the redirect above could be changed to [`/<dest..>`](https://rocket.rs/v0.4/guide/requests/#multiple-segments) to avoid special-casing PDFs here?

Reply:
I took that route as the `<dest>` parameter has pretty particular parsing behaviour, currently only dealing with files. I'd like to rewrite the redirect functionality anyways.

Reply:
https://github.com/rust-lang/www.rust-lang.org/issues/767